packages/x-tree-view-pro/package.json

```diff
@@ -62,7 +62,7 @@
     "react-dom": "^17.0.0 || ^18.0.0"
   },
   "devDependencies": {
-    "@mui/internal-test-utils": "1.0.0",
+    "@mui/internal-test-utils": "^1.0.1",
```

Comment:
Do you know why we define this one in each package and not just at the root?

Reply:
Maybe because `mui/material-ui` has it set up like that... 🤷 🙈 
Not sure, I also don't see too much point in that.
Maybe @michaldudak could clarify as he introduced it with https://github.com/mui/mui-x/pull/12880.

I've pushed a commit keeping only root dependency to see if everything sill works after that: https://github.com/mui/mui-x/pull/13479/commits/7a74141e6a53645c32bce421e794808257587ff0.

Reply:
Because it's used in each of these packages. It may not be a big deal for devDependencies, but it's a good practice to define dependencies where they are used. This way packages can be more self-sustained and not implicitly depend on anything around them.

Reply:
I appreciate the clarification.
Personally, I don't have a strong preference on this, but I understand the position, especially if at some point the package would have to be moved from the monorepo context, there would be one less thing to adjust.
WDYT @flaviendelangle?

Reply:
That's fine for me